accessibility/introduction.md

```diff
@@ -6,11 +6,8 @@
 - [VPATs](#vpats)
 - [How we conform](#how-we-conform)
   - [Standards](#standards-we-aim-for)
-  - [Tools and techniques](#tools-and-techniques)
+  - [Techniques](#techniques)
     - [Guidelines](#guidelines)
-    - [Pa11y](#pa11y)
-    - [Other accessibility testing tools](#other-accessibility-testing-tools)
-    - [Assistive technology](#assistive-technology)
     - [Testing with real users](#testing-with-real-users)
```

Comment:
under `Testing with real users` there is now a `Tools` section that links off to the new tools page. Should this section be linked to from here?